plugins/poll/assets/javascripts/lib/rich-editor-extension.js

```diff
@@ -29,6 +29,7 @@ const extension = {
             chartType: dom.getAttribute("data-poll-chart-type"),
```

Comment:
Above here you need a `dynamic: { default: null },` in the `attrs` section otherwise the `dynamic` attr is lost switching from markdown -> rich